libs/GameEntityManipulation/CreateEntitiesFunctions/CreateBackground.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** CreateBackground
+*/
+
+#ifndef CREATEBACKGROUND_HPP_
+#define CREATEBACKGROUND_HPP_
+
+#include "GameComponents/CollidableComponent.hpp"
+#include "GameComponents/DamageComponent.hpp"
+#include "GameComponents/BackGroundComponent.hpp"
+#include "GameComponents/PositionComponent.hpp"
+#include "GameComponents/SizeComponent.hpp"
+#include "GameComponents/VelocityComponent.hpp"
+#include "GameComponents/WeightComponent.hpp"
+#include "World/World.hpp"
+
+namespace ecs
+{
+    /// @brief This function creates a new BackGround Entity with these parameters
+    /// @param world The world in which the BackGround must be created
+    /// @param pos_x Position x of the BackGround
+    /// @param pos_y Position y of the BackGround
+    /// @param weight Weight of the BackGround
```

Comment:
Why would you add a Weight to the BackGround ?

Reply:
For the second part of the Project with gravity system